### PR TITLE
Add CommandLineUtils to Orbit main

### PR DESCRIPTION
**Pull request by reichlfl**
Required for b/202262468, slight refactoring of utility methods in main.cpp to be able to unit-test them.

If we want to enforce the "Orbit" module to only contain "main.cpp" and nothing else, this could be moved into a different module, but it will be very closely related to how certain parameters are handled in main.cpp (e.g. connection_target in the future).

### CODE REVIEW

File: src/Orbit/CommandLineUtils.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef _ORBIT_MAIN_H__
```
Comment (vickyliu-go4it) [inline src/Orbit/CommandLineUtils.h:5]: It seems that the include guard should be the name of the module followed by the name of the file. So maybe `ORBIT_COMMAND_LINE_UTILS_H_` here.

File: src/Orbit/CommandLineUtils.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#include <string>
+#include <vector>
+
+// Extract command line flags by filtering the positional arguments out from the command line
+// arguments.
+QStringList ExtractCommandLineFlags(const std::vector<std::string>& command_line_args,
+                                    const std::vector<char*>& positional_args);
+QStringList RemoveFlagsNotPassedToMainWindow(const QStringList& flags);
+
+#endif
```
Comment (vickyliu-go4it) [inline src/Orbit/CommandLineUtils.h:18]: Nit: `#endif // ORBIT_COMMAND_LINE_UTILS_H_`

**Review by vickyliu-go4it — APPROVED** [review]
LGTM! Only minor things about the include guard. Thanks for making the change

File: src/Orbit/CommandLineUtils.h (hunk 1 above)
Comment (reichlfl) [reply src/Orbit/CommandLineUtils.h]: Thanks, I've missed this after renaming.

**Comment by reichlfl** [comment]
After talking to @beckerhe, I've moved this into a separate module

**Review by beckerhe — APPROVED** [review]
LGTM

**Pull request merged**